### PR TITLE
Feature/update materialization

### PR DESCRIPTION
**Pull request by jthandy**

### CODE REVIEW

**Comment by jthandy** [comment]
@drewbanin take a look when you have a sec. i've run this in one project and it works.

File: models/events/snowplow_enriched_events.sql [hunk 1]
```diff
@@ -1,8 +1,9 @@
 
 {{
     config({
+        "materialized" : "incremental",
         "distkey"  : "domain_userid",
-        "sortkey"  : ["domain_userid", "domain_sessionidx", "dvce_created_tstamp"],
+        "sortkey"  : ["session_start_tstamp"],
```
Comment (drewbanin) [inline models/events/snowplow_enriched_events.sql]: is `session_start_tstamp` defined in this model?

**Review by drewbanin — COMMENTED** [review]
one comment, otherwise LGTM

**Comment by jthandy** [comment]
ok @drewbanin fixed that comment; merging.

**Pull request merged**